### ISSUE TITLE
The bottom item of right nav can not be actived.

### ISSUE THREAD

**Issue by hhao1** [patternize/patternize.io#10]
When you click the bottom item of the quick access bar, it will jump to the specific topic. but the nav itself won't be activated.
Minor issue.
I guess it is an issue in Docusaurus 2. when the content is not centered, it won't activate the nav item.

![submenu](https://user-images.githubusercontent.com/35816566/72392770-b3ca1f00-36e5-11ea-8a2d-e88665adc987.PNG)
![submenu2](https://user-images.githubusercontent.com/35816566/72392787-bcbaf080-36e5-11ea-873f-4c73a81ead90.PNG)

**Comment by gazcn007** [comment]
yes it's an known issue with Docusaurus 2, I can repro with CRA's doc
![image](https://user-images.githubusercontent.com/12766318/72405845-213e7580-370f-11ea-8d65-f87be3aaef3f.png)

We can always swizzle the theme components and control them with better precision

**Comment by gazcn007** [comment]
Another styling issue, when the website is too narrow, the navbar item padding is weird
![image](https://user-images.githubusercontent.com/12766318/73637169-9b28a580-46a2-11ea-8397-a8f20442da99.png)
